spec/requests/oauth2_spec.rb

```diff
@@ -2,10 +2,38 @@
   let!(:login) { create(:login) }
 
   describe 'POST /token' do
+    before(:all) do
+      @orig_force_ssl = RailsApiAuth.force_ssl
+      RailsApiAuth.force_ssl = false
+    end
```

Comment:
I found [this](https://github.com/rspec/rspec-rails/issues/616#issuecomment-9198768) and just updated the specs...